deploy/src/deploymentUtils.js

```diff
@@ -147,13 +147,128 @@ function logValidatorsAndRewardAccounts(validators, rewards) {
   })
 }
 
+async function upgradeProxy({ proxy, implementationAddress, version, nonce, url }) {
+  const data = await proxy.methods.upgradeTo(version, implementationAddress).encodeABI()
+  const result = await sendRawTxHome({
+    data,
+    nonce,
+    to: proxy.options.address,
+    privateKey: deploymentPrivateKey,
+    url
+  })
+  if (result.status) {
+    assert.strictEqual(Web3Utils.hexToNumber(result.status), 1, 'Transaction Failed')
+  } else {
+    await assertStateWithRetry(proxy.methods.implementation().call, implementationAddress)
+  }
+}
+
+async function transferProxyOwnership({ proxy, newOwner, nonce, url }) {
+  const data = await proxy.methods.transferProxyOwnership(newOwner).encodeABI()
+  const result = await sendRawTxHome({
+    data,
+    nonce,
+    to: proxy.options.address,
+    privateKey: deploymentPrivateKey,
+    url
+  })
+  if (result.status) {
+    assert.strictEqual(Web3Utils.hexToNumber(result.status), 1, 'Transaction Failed')
+  } else {
+    await assertStateWithRetry(proxy.methods.proxyOwner().call, newOwner)
+  }
+}
+
+async function transferOwnership({ contract, newOwner, nonce, url }) {
+  const data = await contract.methods.transferOwnership(newOwner).encodeABI()
+  const result = await sendRawTxForeign({
+    data,
+    nonce,
+    to: contract.options.address,
+    privateKey: deploymentPrivateKey,
+    url
+  })
+  if (result.status) {
+    assert.strictEqual(Web3Utils.hexToNumber(result.status), 1, 'Transaction Failed')
+  } else {
+    await assertStateWithRetry(contract.methods.owner().call, newOwner)
+  }
+}
+
+async function setBridgeContract({ contract, bridgeAddress, nonce, url }) {
+  const data = await contract.methods.setBridgeContract(bridgeAddress).encodeABI()
+  const result = await sendRawTxForeign({
+    data,
+    nonce,
+    to: contract.options.address,
+    privateKey: deploymentPrivateKey,
+    url
+  })
+  if (result.status) {
+    assert.strictEqual(Web3Utils.hexToNumber(result.status), 1, 'Transaction Failed')
+  } else {
+    await assertStateWithRetry(contract.methods.bridgeContract().call, bridgeAddress)
+  }
+}
+
+async function initializeValidators({
+  contract,
+  isRewardableBridge,
+  requiredNumber,
+  validators,
+  rewardAccounts,
+  owner,
+  nonce,
+  url
+}) {
+  let data
+
+  if (isRewardableBridge) {
+    console.log(`REQUIRED_NUMBER_OF_VALIDATORS: ${requiredNumber}, HOME_VALIDATORS_OWNER: ${owner}`)
```

Comment:
Since this functionality could be used to initialize validators in the Foreign chain too it makes sense to remove the prefix `HOME_`. The same is applicable to the similar log message bellow.

Reply:
Thanks! Fixed here https://github.com/poanetwork/poa-bridge-contracts/pull/156/commits/d92dd655faae5068c355551a7b5893ba90afb54d